src/partials/config.html

```diff
@@ -1,30 +1,30 @@
 <h3 class="page-heading">OCI settings</h3>
 
 <div class="gf-form-group">
-<div class="gf-form max-width-30">
-  <label class="gf-form-label width-10">Tenancy OCID</label>
+  <div class="gf-form max-width-30">
+    <label class="gf-form-label width-10">Environment</label>
+    <select
+      class="gf-form-input"
+      ng-model="ctrl.current.jsonData.environment"
+	    ng-required="true"
+      ng-options="o as o for o in ctrl.getEnvironments()" >
+    </select>   
+  </div>
+  <div class="gf-form max-width-30">
+  <label class="gf-form-label width-10" ng-if="ctrl.current.jsonData.environment!=='multitenancy'">Tenancy OCID</label>
     <input class="gf-form-input " 
 		type="text" 
 		ng-required="true"
-    ng-model="ctrl.current.jsonData.tenancyOCID">
+    ng-model="ctrl.current.jsonData.tenancyOCID" ng-if="ctrl.current.jsonData.environment!=='multitenancy'">
   </input>
   </div>
   <div class="gf-form max-width-30">
-    <label class="gf-form-label width-10">Default Region</label>
+    <label class="gf-form-label width-10" ng-if="ctrl.current.jsonData.environment!=='multitenancy'">Default Region</label>
```

Comment:
Without the default region setting in the multitenancy case, how does the datasource test function know what region to use?

Reply:
default region restored